### PR TITLE
24/7 Deathmatch

### PR DESCRIPTION
**Pull request by N3X15**
Turn /vg/ into what it always wanted to be, a 24/7 highlander server.

* Forces every round to extended.
* Starts highlander 10s afterwards.

### CODE REVIEW

File: code/controllers/configuration.dm [hunk 1]
```diff
@@ -724,11 +723,13 @@
 /datum/configuration/proc/pick_mode(mode_name)
 	// I wish I didn't have to instance the game modes in order to look up
 	// their information, but it is the only way (at least that I know of).
+	/*
```
Comment (DamianX) [inline code/controllers/configuration.dm]: dead code, remove it

**Review by DamianX — CHANGES_REQUESTED** [review]

**Comment by ultraplen** [comment]
bbc5b63c-17c7-11e7-8929-b083b7a9ddcd

idk i dont this pr would be good for the server culture

**Comment by N3X15** [comment]
The culture of silent murdergangs roaming the station, hunting for even the slightest hint of fun?

**Review by PJB3005 — APPROVED** [review]
Getting closer to the authentic TG experience every day.

**Comment by DrSnips** [comment]
Merge when?

**Comment by ultraplen** [comment]
the murdergangs are very different from just straight up highlander because murdergangs need to rp for a little bit before they lynch someone

**Comment by N3X15** [comment]
Since when have they ever RPed to smash someone's face in with a toolbox?

**Comment by ghost** [comment]
Needs to play https://www.youtube.com/watch?v=ciPu4DnKBuM in a loop

**Comment by ultraplen** [comment]
Although its anecdotal, whenever I shuffle down the main hallway, with a crossbow in one hand and a stunprod in the other, I usually have say"HANDS UP RIGHT NOW OR ILL KILL YOU" pretyped so that I can say it before shooting someone.

**Comment by ihadtoregisterforthis** [comment]
:+1: Finally a step in the right direction

**Comment by Aurx** [comment]
I'm confused.  
N3X15, why did you write a PR that does nothing at all?

**Comment by Zennyization** [comment]
What is wrong with you.

File: code/controllers/configuration.dm [hunk 2]
```diff
@@ -722,13 +721,6 @@
 				diary << "Unknown setting in configuration: '[name]'"
 
 /datum/configuration/proc/pick_mode(mode_name)
-	// I wish I didn't have to instance the game modes in order to look up
-	// their information, but it is the only way (at least that I know of).
```
Comment (Exxion) [inline code/controllers/configuration.dm:726]: Looks like someone didn't know their undocumented BYOND "features" very well

**Comment by ShiftyRail** [comment]
Holy fuck he did it the absolute madman

**Comment by aceedex** [comment]
Nice meme lad

**Comment by DrSnips** [comment]
We did it reddit.

**Comment by WhiteHusky** [comment]
**_`THE MEMING BEGINS AT FULL FORCE`_**

**Comment by WhiteHusky** [comment]
The meming has stopped.
#16241

**Comment by Exxion** [comment]
Holy shit what the fuck

**Pull request merged**